doc/api_queries.rst

```diff
@@ -60,11 +60,15 @@
 
   Notice that none of the filtering methods mutates the state of the :class:`~dnf.query.Query` but produces a new object instead.
 
-  .. method:: available
+  .. method:: available()
 
-    Return a new query limiting the original query to the not-installed packages, that is packages available from the repositories.
+    Return a new query limiting the original query to the packages available from the repositories.
```

Comment:
Probably a good idea to unify the 3rd person usage (Return vs Returns) throughout the file?